executor/src/main/scala/org/apache/amaterasu/executor/execution/actions/runners/spark/PySpark/PySparkRunner.scala

```diff
@@ -16,8 +16,9 @@
  */
 package org.apache.amaterasu.executor.execution.actions.runners.spark.PySpark
 
-import java.io.{File, PrintWriter, StringWriter}
+import java.io._
```

Comment:
As a convention, it is recommended to avoid import entire libraries (as in python it is kind of a code-smell to do 'from foo import *'. Please specify specific classes/objects etc to import.